### DISCUSSION TITLE
关于参考文献序号是左对齐还是右对齐？

### DISCUSSION THREAD

**Discussion by lingr7** [nuaatug/nuaathesis#76] [Q&A] [answered]
![Snipaste_2021-12-01_08-39-44](https://user-images.githubusercontent.com/37898999/144150979-6e06d591-b50e-4581-98a5-2daeb0e02a10.png)
以上是左对齐。
现在模板是右对齐，如何在nuaabib.bst中修改为左对齐。
![Snipaste_2021-12-01_08-40-51](https://user-images.githubusercontent.com/37898999/144151069-b1dadd1e-4b29-4e8f-a36d-b1d9ce866e89.png)

**Comment by yzwduck** [comment, marked as answer]
"global.tex" 里添加
```latex
\renewcommand{\bibnumfmt}[1]{\hfill{}[#1]}
```

**Reply by lingr7** [reply]
感谢回复，经过实验
```tex
\renewcommand{\bibnumfmt}[1]{[#1]\hfill{}}
```
这条命令可以做到左对齐。
之前的那个命令，在左边添加了水平间距，但是视觉效果依然是右对齐。
![Snipaste_2021-12-01_23-34-37](https://user-images.githubusercontent.com/37898999/144264180-99ad5933-b6ff-4e7b-9be9-b7c71482c687.png)
